apps/wallet-mobile/src/i18n/LanguageProvider.tsx

```diff
@@ -1,4 +1,6 @@
-import {parseSafe, useStorage} from '@yoroi/common'
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Maybe it's better to just disable the check on the line instead of the entire file

Reply:
it doesn't work for the `ban-ts-comment` cuz is inside the JSX component, it keeps complaining that's why it was made for the file